components/lcd/esp_lcd_gc9503/idf_component.yml

```diff
@@ -0,0 +1,11 @@
+version: "1.0.0"
+description: ESP LCD GC9503
+url: https://github.com/espressif/esp-bsp/tree/master/components/lcd/esp_lcd_gc9503
+dependencies:
+  idf: ">=4.4"
+  esp_io_expander:
+    version: "^1"
+    public: true
```

Comment:
I don't see where this compont is used, or why it is needed ?

Reply:
Sorry, it's a mistake. I also remove the `esp_lcd_panel_io_additions` which is unnecessary.